src/jsonUtils/computeYogaNode.js

```diff
@@ -0,0 +1,336 @@
+  const style: ViewStyle | Object = hasStyle ? getStyles(node) : {};
+
+  // Setup default symbol instance dimensions
+  if (node.type === 'symbolinstance') {
+    const symbolProps = node.props;
+    const { frame } = getSymbolMasterByName(symbolProps.masterName);
+    yogaNode.setWidth(frame.width);
+    yogaNode.setHeight(frame.height);
+  }
+
+  if (hasStyle) {
+    // http://facebook.github.io/react-native/releases/0.48/docs/layout-props.html
+
+    // Width
+    if (!isNullOrUndefined(style.width)) {
+      yogaNode.setWidth(style.width);
+    }
+
+    // Height
+    if (!isNullOrUndefined(style.height)) {
+      yogaNode.setHeight(style.height);
+    }
+
+    // Min-Height
+    if (!isNullOrUndefined(style.minHeight)) {
+      yogaNode.setMinHeight(style.minHeight);
+    }
+
+    // Min-Width
+    if (!isNullOrUndefined(style.minWidth)) {
+      yogaNode.setMinWidth(style.minWidth);
+    }
+
+    // Max-Height
+    if (!isNullOrUndefined(style.maxHeight)) {
+      yogaNode.setMaxHeight(style.maxHeight);
+    }
+
+    // Min-Width
+    if (!isNullOrUndefined(style.maxWidth)) {
+      yogaNode.setMaxWidth(style.maxWidth);
+    }
+
+    // Margin
+    if (!isNullOrUndefined(style.marginTop)) {
+      yogaNode.setMargin(yoga.EDGE_TOP, style.marginTop);
+    }
+    if (!isNullOrUndefined(style.marginBottom)) {
+      yogaNode.setMargin(yoga.EDGE_BOTTOM, style.marginBottom);
+    }
+    if (!isNullOrUndefined(style.marginLeft)) {
+      yogaNode.setMargin(yoga.EDGE_LEFT, style.marginLeft);
+    }
+    if (!isNullOrUndefined(style.marginRight)) {
+      yogaNode.setMargin(yoga.EDGE_RIGHT, style.marginRight);
+    }
+    if (!isNullOrUndefined(style.marginVertical)) {
+      yogaNode.setMargin(yoga.EDGE_VERTICAL, style.marginVertical);
+    }
+    if (!isNullOrUndefined(style.marginHorizontal)) {
+      yogaNode.setMargin(yoga.EDGE_HORIZONTAL, style.marginHorizontal);
+    }
+    if (!isNullOrUndefined(style.margin)) {
+      yogaNode.setMargin(yoga.EDGE_ALL, style.margin);
+    }
+
+    // Padding
+    if (!isNullOrUndefined(style.paddingTop)) {
+      yogaNode.setPadding(yoga.EDGE_TOP, style.paddingTop);
+    }
+    if (!isNullOrUndefined(style.paddingBottom)) {
+      yogaNode.setPadding(yoga.EDGE_BOTTOM, style.paddingBottom);
+    }
+    if (!isNullOrUndefined(style.paddingLeft)) {
+      yogaNode.setPadding(yoga.EDGE_LEFT, style.paddingLeft);
+    }
+    if (!isNullOrUndefined(style.paddingRight)) {
+      yogaNode.setPadding(yoga.EDGE_RIGHT, style.paddingRight);
+    }
+    if (!isNullOrUndefined(style.paddingVertical)) {
+      yogaNode.setPadding(yoga.EDGE_VERTICAL, style.paddingVertical);
+    }
+    if (!isNullOrUndefined(style.paddingHorizontal)) {
+      yogaNode.setPadding(yoga.EDGE_HORIZONTAL, style.paddingHorizontal);
+    }
+    if (!isNullOrUndefined(style.padding)) {
+      yogaNode.setPadding(yoga.EDGE_ALL, style.padding);
+    }
+
+    // Border
+    if (!isNullOrUndefined(style.borderTop)) {
```

Comment:
I think these are actually `borderTopWidth` etc. And apparently, there's no support for `borderHorizontalWidth` or `borderVerticalWidth` (but margin and padding *do* have these). Go figure!

Reply:
Hmm..weird. I'm going off of the Node.h file from the yoga JS source (https://github.com/facebook/yoga/blob/master/javascript/sources/Node.hh#L102). Seems as though there is only `setBorder(int edge, double border);` available. Is there better information on this elsewhere?

Reply:
I meant the styles---the yoga stuff is fine. http://facebook.github.io/react-native/releases/0.48/docs/layout-props.html#borderbottomwidth

Reply:
Ahh, good call. Sorry for the misunderstanding. Should be fixed now